tests/Rector/v12/v0/MoveAllowTableOnStandardPagesToTCAConfigurationRector/config/configured_rule.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+declare(strict_types=1);
+
+use Rector\Config\RectorConfig;
+use Ssch\TYPO3Rector\TYPO312\v0\IgnorePageTypeRestrictionRector;
+use Ssch\TYPO3Rector\TYPO312\v0\MoveAllowTableOnStandardPagesToTCAConfigurationRector;
+
+return static function (RectorConfig $rectorConfig): void {
+    $rectorConfig->import(__DIR__ . '/../../../../../../config/config.php');
+    $rectorConfig->ruleWithConfiguration(IgnorePageTypeRestrictionRector::class, []);
```

Comment:
how does the test actually work without a table configuration?

Reply:
I have removed IgnorePageTypeRestrictionRector. It is not in use anymore. It was before in my draft. Good catch